src/common/QskRgbValue.h

```diff
@@ -235,4 +235,136 @@ namespace QskRgb
 
 #endif
 
+namespace QskRgb
+{
+    // Converts a HTML #RRGGBB[AA] color hex string to Qt's RGB 0xAARRGGBB integer
+    QSK_EXPORT Q_REQUIRED_RESULT constexpr QRgb fromHexString(
+        const char* const str, const size_t len ) noexcept
+    {
+        if ( len != 7 && len != 9 )
+        {
+            return 0;
+        }
+
+        if ( str[ 0 ] != '#' )
+        {
+            return 0;
+        }
+
+        QRgb rgb = 0x00;
```

Comment:
```
In file included from common/QskRgbValue.cpp:6:
common/QskRgbValue.h:255:14: warning: variable declaration in a constexpr function is a C++14 extension [-Wc++14-extensions]
        QRgb rgb = 0x00;
             ^
```

Reply:
see #287 - will do this today.

Reply:
done: using c++17 features is allowed now